src/plugins/workspace/public/components/workspace_form/constants.ts

```diff
@@ -138,7 +138,7 @@ export const permissionModeOptions = [
     inputDisplay: i18n.translate(
       'workspace.form.permissionSettingPanel.permissionModeOptions.readAndWrite',
       {
-        defaultMessage: 'Read & Write',
+        defaultMessage: 'Read and write',
```

Comment:
Nit: can we use `WORKSPACE_ACCESS_LEVEL_NAMES` in `src/plugins/workspace/public/constants.ts` instead?